tlsn/tlsn-notary-client/src/client.rs

```diff
@@ -0,0 +1,336 @@
+//! Notary client
+//!
+//! This module sets up prover to connect to notary via TCP or TLS
+
+use eyre::eyre;
```

Comment:
Is that because this kind of idiomatic error handling tool is for application rather than library; when one doesn't care about error type other than reporting it?